scenes/game/local_game.gd

```diff
@@ -7257,61 +7344,67 @@ func continue_extra_attack(events):
 			printlog("EXTRA ATTACK: Pausing for decision %s %s" % [player_name, Enums.DecisionType.keys()[decision_info.type]])
 			break
 
-		printlog("EXTRA ATTACK: processing state %s " % [ExtraAttackState.keys()[active_strike.extra_attack_state]])
-		match active_strike.extra_attack_state:
+		printlog("EXTRA ATTACK: processing state %s " % [ExtraAttackState.keys()[active_strike.extra_attack_data.extra_attack_state]])
+		match active_strike.extra_attack_data.extra_attack_state:
 			ExtraAttackState.ExtraAttackState_PayCosts:
 				# Ask player to pay for this card if applicable.
 				events += ask_for_cost(attacker_player, attacker_card, ExtraAttackState.ExtraAttackState_DuringStrikeBonuses)
 			ExtraAttackState.ExtraAttackState_DuringStrikeBonuses:
 				events += do_effects_for_timing("during_strike", attacker_player, attacker_card, ExtraAttackState.ExtraAttackState_Activation, true)
 			ExtraAttackState.ExtraAttackState_Activation:
-				active_strike.extra_attack_state = ExtraAttackState.ExtraAttackState_Before
-				active_strike.extra_attack_remaining_effects = get_all_effects_for_timing("before", attacker_player, attacker_card, true, true)
+				active_strike.extra_attack_data.extra_attack_state = ExtraAttackState.ExtraAttackState_Before
+				active_strike.extra_attack_data.extra_attack_remaining_effects = get_all_effects_for_timing("before", attacker_player, attacker_card, true, true)
 			ExtraAttackState.ExtraAttackState_Before:
 				events += do_remaining_effects(attacker_player, ExtraAttackState.ExtraAttackState_DetermineHit)
 			ExtraAttackState.ExtraAttackState_DetermineHit:
 				var hit = determine_if_attack_hits(events, attacker_player, defender_player, attacker_card)
 				if hit:
-					active_strike.extra_attack_hit = true
-					active_strike.extra_attack_state = ExtraAttackState.ExtraAttackState_Hit
-					active_strike.extra_attack_remaining_effects = get_all_effects_for_timing("hit", attacker_player, attacker_card, true, true)
+					active_strike.extra_attack_data.extra_attack_hit = true
```

Comment:
ok, now i see why you put player1_hit in the regular strike thing.
This makes sense since you don't want to make the previous attack count as hit for going to gauge.